spyderlib/widgets/editor.py

```diff
@@ -581,10 +581,16 @@ def open_fileswitcher_dlg(self):
         """Open file list management dialog box"""
         if not self.tabs.count():
             return
+        if self.fileswitcher_dlg is not None and \
+          self.fileswitcher_dlg.is_visible:
+            self.fileswitcher_dlg.hide()
+            self.fileswitcher_dlg.is_visible = False

```

Comment:
Ops, I didn't know about it. I looked for `isVisible` but it seems QDialogs don't support it.

I'll change it in master.